src/models/index.ts

```diff
@@ -0,0 +1,23 @@
+import StateListener from './state-listener';
```

Comment:
Rather than having a separate import and export, can these just be export statements like the other lines?